src/schemas/SettingsSchema.js

```diff
@@ -45,9 +47,18 @@ module.exports = graphql.buildSchema(`
     sites: [Site]!,
   }
 
+  type PageInfo {
+    cursor: String
+  }
+
   type TwitterAccountCollection {
+    accounts: [TwitterAccount]
```

Comment:
I propose we just add `pageInfo: PageInfo` here. As long as it's not required, we should remain backwards-compatible with Fortis v1 UI.